website/pages/en/sunrise.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: Sunrise of decentralized data FAQ
+---
+
+## What is the sunrise of decentralized data?
+
+The sunrise of decentralized data is an initiative spearheaded by Edge & Node, working on The Graph. The goal is to seamlessly enable subgraph developers and data consumers to upgrade to The Graph’s decentralized network.
+
+This plan draws on many previous developments from The Graph ecosystem, including an upgrade Indexer to serve queries on newly published subgraphs, and the ability to integrate new blockchain networks to The Graph.
+
+### What are the phases of the sunrise of decentralized data?
+
+**Sunray**: Enable support for hosted service chains, offer a seamless upgrade flow, offer a free plan on the network  
+**Sunbeam**: The upgrade window that subgraph developers will have to upgrade their subgraphs to The Graph Network. The start and actual length of this will be announced soon.  
+**Sunrise**: Hosted service endpoints will expire as traffic moves to The Graph Network
```

Comment:
Only one of these 3 points has periods at the end of sentences, so maybe the other 2 should as well?

Reply:
And a comment on the content... is it normal that there are no dates at all here, not even estimates, not even for Sunray?

Reply:
Got those periods. Yes, this was intentional not to include dates.